easybuild/easyconfigs/l/lpsolve/lpsolve-5.5.2.5-iccifort-2017.4.196-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CmdCp'
+
+name = 'lpsolve'
+version = '5.5.2.5'
+
+homepage = 'https://sourceforge.net/projects/lpsolve/'
+description = "Mixed Integer Linear Programming (MILP) solver"
+
+toolchain = {'name': 'iccifort', 'version': '2017.4.196-GCC-6.4.0-2.28'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['lp_solve_%(version)s_source.tar.gz']
+patches = ['lpsolve-5.5.2.5_Fix_expected_identifier_error.patch']
```

Comment:
The patch is not needed here since only GCC generates the preprocessor error.

Reply:
Then maybe the patch filename should be changed to clarify it's only required with `GCC`?